plugins/justify/plugin.js

```diff
@@ -178,8 +188,18 @@
 
 		refresh: function( editor, path ) {
 			var firstBlock = path.block || path.blockLimit;
+			var name = firstBlock.getName();
+
+			// Check if we can give to element proper class, style or current path is body (#455).
+			// IE8, after focus and refresh return path with 'body'
+			if ( editor.config.enterMode === CKEDITOR.ENTER_BR && firstBlock === editor.editable() ) {
```

Comment:
To sum up: I think that it'd be better to just enable justify if the whole editable is selected